Targets/LPC23xx_LPC24xx/LPC24_Flash.cpp

```diff
@@ -59,7 +59,7 @@ struct DeploymentState {
 static DeploymentState deploymentStates[TOTAL_DEPLOYMENT_CONTROLLERS];
 
 const char* flashApiNames[TOTAL_DEPLOYMENT_CONTROLLERS] = {
-    "GHIElectronics.TinyCLR.NativeApis.LPC24.StorageController\\0"
+    "GHIElectronics.TinyCLR.NativeApis.LPC24.InternalFlash.StorageController\\0"
```

Comment:
Should be `"GHIElectronics.TinyCLR.NativeApis.LPC24.InternalFlashStorageController\\0"`